### ISSUE TITLE
Duplicate Last.FM Scrobbles

### ISSUE THREAD

**Issue by hurmaqny** [web-scrobbler/web-scrobbler#2863]
I use web-scrobbler to last.fm and libre.fm. Here's how I do that:

- I listen to music via the Spotify desktop app
- I'm signed in to both last and libre fm on the web-sbrobbler app.
- I have the Spotify web player open in Edge

Until some time last week, I had no problems with this setup and all scrobbles appeared correctly. However, something must have changed, not sure if it's web-scrobbler or my browser, or perhaps even Spotify, but now every scrobble to last.fm is duplicated.
![image](https://user-images.githubusercontent.com/4416465/118450859-512fcb80-b6fd-11eb-955e-8ca59f797c32.png)

**Comment by alexesprit** [comment]
Not sure if this an issue. The extension scrobbles music from Spotify even if other players are connected (except Web Player, which is mostly a separate tab in the browser).

If you want to avoid this behavior, you can turn off scrobbling from Spotify via the exension options and continue listening to music via Spotify Desktop.

Probably, it's related to #2820.

**Comment by hurmaqny** [comment]
Okay, but doing that will prevent me from scrobbling to libre.fm and that's one of the main reasons I use web-scrobbler.

**Comment by alexesprit** [comment]
Does Spotify Desktop allow to turn off scrobbling?

**Comment by hurmaqny** [comment]
No, scrobbling happens through a Spotify application that you connect in your Last FM account.

**Comment by alexesprit** [comment]
What solution do you propose?

Personally, I don't want to just disable scrobbling if other connections are available. This way breaks other usage flow.

**Comment by hurmaqny** [comment]
I'm actually not even sure why this is happening now. I used to be able to listen to music using Spotify desktop with Spotify web player open in the web browser, connected to both last and libre fm and not get duplicate scrobbles.

I was actually under the impression that duplication was resolved back here https://github.com/web-scrobbler/web-scrobbler/issues/2242#issuecomment-578474783, because I've always had the Spotify web player open in a tab and was able to scrobble to libre and last fm with no problems.

So, if that's no longer possible, it's not a problem at all. Feel free to close this issue and I will have to figure out another way to scrobble to libre fm.
